pygments_ansi_color/__init__.py

```diff
@@ -122,7 +130,15 @@ def color_tokens(
             'Blue': '#3465A4',
             'Magenta': '#c509c5',
             'Cyan': '#34E2E2',
-            'White': '#ffffff',
+            'White': '#CFCFCF',
+            'BrightBlack': '#676767',
+            'BrightRed': '#FF6D67',
+            'BrightGreen': '#5FF967',
+            'BrightYellow': '#FEFB67',
+            'BrightBlue': '#6871FF',
+            'BrightMagenta': '#FF76FF',
+            'BrightCyan': '#5FFDFF',
+            'BrightWhite': '#FEFFFF',
```

Comment:
Can you add a screenshot of what these new colors look like? I'd like to make sure they look consistent with the existing default colors.

Also, was changing white intentional?

Reply:
Bright colors:
<img width="353" alt="image" src="https://user-images.githubusercontent.com/1004907/220446906-d84955b9-d7c1-4fb6-9beb-df60be2af4be.png">

These are loosely based on the color scheme used by iterm2.

> Also, was changing white intentional?

Yes. If you have a bright white, the regular white cannot be as bright.

Reply:
I re-created the original testing image that I used for this repo with bright colors added:

![Screenshot_2023-02-21_21-15-19](https://user-images.githubusercontent.com/665269/220512247-5c6957e9-f386-45a9-9c64-fe01473329d0.png)

(Rows 3, 6, 9 have bright FG colors; rows 7-9 have bright BG colors.)

The bright colors look good to me but I think the grayer white is a bit of a regression since white FG with colored background is a pretty common scenario and it is harder for me to read with some of the backgrounds (especially green and red).

How do you feel about `#F5F5F5` for the default white? I think it looks OK, compare what you have in this PR (`#CFCFCF`):
![have](https://user-images.githubusercontent.com/665269/220513927-1a086e78-4d8c-4feb-a10c-20171717d3b8.png)


vs my proposal (`#F5F5F5`):

![proposal](https://user-images.githubusercontent.com/665269/220513919-09ecc8c5-b96b-43dc-ab89-0b61eade3e24.png)


The "bright" white is still a bit brighter. Not quite as noticeable but I think it's worth it for the readability?

Reply:
I have changed it to #f5f5f5 in 3dc65ca. I have no problems with changing the defaults for any of these colours, as they are only in comments and documentation anyway, and I'll override them (and the default colors as well) in the applications I'll use this in.